amundsenatlastypes/schema/01_2_table_schema.json

```diff
@@ -23,6 +23,15 @@
           "isUnique": false,
           "isIndexable": true,
           "includeInNotification": false
+        },
```

Comment:
Do you need this here ?

Reply:
If I don't have this relationship then I wouldn't be able to acquire table details if I wanted (like description)

Reply:
In that case, you need to create a relationship attribute between Table and Bookmark. 
So that by simply getting the bookmark, you'd be able to get the Table as well. 

Bookmark:
- active
- entityUri
- entity/Table/parentEntity etc.

Reply:
I understood that we need to capture every `User` relationship, not every `Table` relationship.

Reply:
what about relationship to User ? I thought we aim for capturing all the User related relationships

Reply:
ah yes that too. 

Bookmark:
active
entityUri
entity/Table/parentEntity etc.
user

Reply:
so I think relationships should be defined the same way they are for the Reader (for consistency). I will do it like that and push changes once verified

Reply:
so there will be two additional relationships?
Bookmark <-> Table 
Bookmark <-> User 

is that possible to bookmark anything apart from Table ?

Reply:
the plan is to have User <> Bookmark <> Referenceable

Reply:
added two relationships to Bookmark - one for User another one for Referenceable. That gives us possibility to bookmark different kinds of entities.